medusa/tv/episode.py

```diff
@@ -1077,11 +1078,11 @@ def to_json(self, detailed=True):
         data['airDate'] = self.air_date
         data['title'] = self.name
         data['description'] = self.description
-        data['content'] = []
+        data['content'] = {}
```

Comment:
Can you put this below, where you set the content? (line 1104)

Reply:
Sure